app/views/qsets/_qset_list.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="row qset_list">
+  <% @qsets.each_with_index do |qset, index| %>
+    <div class="panel panel-default">
+      <div class="panel-heading">
+        <h4 class="panel-title">
```

Comment:
Seems like we're using `.panel-title` inside `.panel-heading` inside `.panel` all over the place, but we're using it like the contents of a panel, not the heading. @sarahschachman we should look at simplifying this to the [basic panel use case](http://getbootstrap.com/components/#panels-basic) in future CSS updates/code reviews, and instead use our own styling to get the look we want.